stdlib/string.gr

```diff
@@ -351,17 +415,16 @@ export let rec implode = (arr: Array<Char>) => {
   let mut stringByteLength = 0n
 
   for (let mut i = 0n; i < arrLength; i += 1n) {
-    let char = WasmI32.load(arr + (i << 2n), 8n)
-    let byte = WasmI32.load8U(char, 4n)
+    let usv = WasmI32.load(arr + (i << 2n), 8n) >>> 3n
 
-    let n = if ((byte & 0x80n) == 0x00n) {
+    let n = if (usv <= 0x80n) {
```

Comment:
This should be `let n = if (usv <= 0x7Fn) {`. Code point 128 gets encoded as two bytes in UTF-8.